packages/slice-machine/components/Badge/StatusBadgeWithTooltip.tsx

```diff
@@ -0,0 +1,61 @@
+import React from "react";
+import ReactTooltip from "react-tooltip";
+import { Badge, Text } from "theme-ui";
+import { FrontEndCustomType } from "../../src/modules/availableCustomTypes/types";
+import { CustomTypeStatus } from "../../src/modules/selectedCustomType/types";
+
+const StatusEnumToDisplayNameAndTooltip = (status: CustomTypeStatus) => {
+  switch (status) {
+    case CustomTypeStatus.New:
+      return {
+        statusDisplayName: "New",
+        statusTooltip:
+          "This Custom Type exists only locally. Upon sync, it will be pushed to your remote repository.",
+      };
+    case CustomTypeStatus.Modified:
+      return {
+        statusDisplayName: "Modified",
+        statusTooltip:
+          "This Custom Type has been modified locally. Upon sync, changes will be pushed to your remote repository.",
+      };
+    case CustomTypeStatus.Synced:
+      return {
+        statusDisplayName: "Synced",
+        statusTooltip:
+          "This Custom Type is in sync with the remote repository.",
+      };
+    case CustomTypeStatus.Unknown:
+      return {
+        statusDisplayName: "Unknown",
+        statusTooltip:
+          "Data from the remote repository could not be fetched (unknown error).",
+      };
+  }
+};
+
+export const StatusBadgeWithTooltip = ({
+  customType,
+}: {
+  customType: FrontEndCustomType;
```

Comment:
can you change this into an interface and use the `StatusBadgeWithTooltip: React.FC<StatusBadgeWithTooltip> = ` instead pls? Just for consistency

Reply:
...like `const StatusBadgeWithTooltip: React.FC<{customType: FrontEndCustomType> => ....` ?

Reply:
<img width="777" alt="Screenshot 2022-08-04 at 13 47 04" src="https://user-images.githubusercontent.com/69536010/182850590-1b19073f-38aa-4502-adf2-b6082151d6f3.png">